src/main/java/uk/gov/digital/ho/pttg/api/HmrcResource.java

```diff
@@ -73,7 +74,8 @@ private IncomeSummary produceIncomeSummary(Individual individual, LocalDate from
 
         log.info("Income summary successfully retrieved from HMRC",
                 value(EVENT, HMRC_SERVICE_RESPONSE_SUCCESS),
-                value(REQUEST_DURATION_MS, requestHeaderData.calculateRequestDuration())
+                value(REQUEST_DURATION_MS, requestHeaderData.calculateRequestDuration()),
+                value(THREAD_COUNT, requestHeaderData.threadCount())
```

Comment:
As discussed, I think this line is superfluous - the changes to the MDC in RequestHeaderData will add the thread count to all logs.